src/lamplib/src/genny/curator.py

```diff
@@ -178,9 +178,7 @@ class CuratorDownloader(Downloader):
     # These build IDs are from the Curator Evergreen task.
     # https://evergreen.mongodb.com/waterfall/curator
 
-    # Note that DSI also downloads Curator, the location is specified in defaults.yml.
```

Comment:
I removed this note about DSI, since the use of curator in DSI is deprecated with perf.send. I think later on in the Cedar migration project it'll behoove us to remove it from DSI entirely.